R/zchunk_L2521.MACC_IRR.R

```diff
@@ -36,8 +36,8 @@ module_emissions_L2521.MACC_IRR <- function(command, ...) {
     # Load required inputs
     A_MACC_TechChange <- get_data(all_data, "emissions/A_MACC_TechChange") %>%
       rename(tech.change = tech_change)
-    L252.AgMAC <- get_data(all_data, "temp-data-inject/L252.AgMAC")
-    L252.MAC_an <- get_data(all_data, "temp-data-inject/L252.MAC_an")
```

Comment:
Thanks for doing this. Did you remove corresponding files from `temp-data-inject`?

Reply:
Done